src/catalogue/category/catalogueCard.component.tsx

```diff
@@ -33,10 +39,17 @@ function CatalogueCard(props: CatalogueCardProps) {
     onToggleSelect(catalogueCategory);
   };
 
+  const handleActionsClose = () => {
+    setMenuOpen(false);
+  };
+
+  const [menuOpen, setMenuOpen] = React.useState<boolean>(false);
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+
   return (
     <Button
       component={Link}
-      to={`${catalogueCategory.id}`}
+      to={menuOpen ? '' : `${catalogueCategory.id}`}
```

Comment:
```suggestion
      to={catalogueCategory.id}
```
I'm not keen on doing this as it doesn't prevent it from trying to route to `''`, instead you should be able to use `event.preventDefault` on the click event for the menu itself like is done for the checkbox below to prevent the events from propagating to the card action component.

Reply:
I added this as if you click on the horizon icon, then click elsewhere on the screen to close it (as if you mis clicked on it) It would then route to the category you clicked the horizon icon of. I do use `event.preventDefault` when clicking on the horizon icon (and on edit, delete, save as). But the behaviour still occurs

Reply:
I believe this is occurring as the prevent default needs to be in the `<Menu>` component itself, I tried changing it and it fixed it for me, the opened menu is effectively delaying the routing until the dialog is closed.